articles/TOC.yml

```diff
@@ -1,13 +1,14 @@
 - name: Welcome
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
 - name: Quantum computing concepts
```

Comment:
Check my recent PR (#523).
Looks like it could be dropped in directly or adapted to the "items: ... - name: ..." format you used just below (which appears much simpler, but I didn't know existed--still figuring out some of the Markdown details).